charts/bulk-scan-processor/values.preview.template.yaml

```diff
@@ -19,31 +19,21 @@ java:
       secretRef: storage-secret-{{ .Release.Name }}
       key: accessKey
   environment:
-    STORAGE_BLOB_SIGNATURE_ALGORITHM: "sha256withrsa"
-    STORAGE_BLOB_PUBLIC_KEY: "nonprod_public_key.der"
     STORAGE_URL: "https://$(STORAGE_ACCOUNT_NAME).blob.core.windows.net"
-    STORAGE_BLOB_SELECTED_CONTAINER: "ALL"
```

Comment:
was this meant to be deleted?

Reply:
yes added too `values.yaml` as we will have this default for all envrs